hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.module.scss

```diff
@@ -5,22 +5,56 @@
     &Btn,
     &IconBtn {
         color: color(white);
-
         .customIcon {
             width: 20px;
+            fill: color(white);
         }
-
         &:hover {
             background-color: rgba(355, 355, 355, 0.2);
         }
     }
-
     &Btn {
         border-radius: 0;
         padding: 20px 15px;
     }
-
     &IconBtn {
         margin-left: 15px;
     }
 }
+
+.navMobile {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    &Btn {
+        height: 50px;
+        width: 100%;
+        display: flex;
+        justify-content: left;
+        padding: 0px 15px;
```

Comment:
Minor, but my IDE yells when you specify a unit after 0, since you don't need to. There's a lot of places we already do this though.